frontend/src/lib/lemon-ui/icons/icons.stories.tsx

```diff
@@ -42,11 +47,33 @@ When adding new icons from Figma please make sure to:
 
 export function Library(): JSX.Element {
     const [showBorder, setShowBorder] = React.useState(true)
+    const [search, setSearch] = React.useState('')
+    const [filteredIcons, setFilteredIcons] = React.useState(allIcons)
+
+    const fuse = new Fuse(allIcons, {
+        keys: ['name'],
+        threshold: 0.3,
+    })
```

Comment:
nit: this could go out of the function, as `allItems` is const and the fuse index doesn't need to be rebuilt on each render.